app/controllers/application_controller.rb

```diff
@@ -148,8 +154,21 @@ def merge_last_user(attrs)
 
   protected
 
-  def set_entity
-    @entity = Entity.find(params[:id])
+  def set_entity(args = {})
+    @skope = Entity.unscoped.includes(args[:includes])
```

Comment:
I like that ` set_entity_with_eager_loading ` is being adapted to use ` set_entity ` that's very smart.

However I think there's a bug here.  includes()[ accepts *args](https://apidock.com/rails/ActiveRecord/QueryMethods/includes) and i think the way that it's being called in the entities_controller will ignore some of the loaded associations.

here' what i mean. 

The original implementation of set_entity_with_eager_loading with this:
``` rb
 @entity = Entity.includes(:aliases, list_entities: [:list]).find(params[:id])
```
which has been changed to this:
```
set_entity(includes: :aliases, list_entities: [:list]) 
```

When the `set_entity ` uses args[:includes] which will only expand to: 
` Entity.scoped.includes(:aliases) `  instead of what we want: ` Entity.scoped.includes(:aliases, list_entities: [:list]) `

Does that make sense? 

This can be trick to catch in tests since if you don't eagar load associations you get longer request times (more db calls) but not an error.

Reply:
yup. had a hunch i'd gotten the signature wrong here. will fix. not sure what to put in place to make sure this doesn't regress. but will make sure it's doing the loading properly